lib/active_fedora/orders/association.rb

```diff
@@ -82,6 +82,12 @@ def delete_at(loc)
       target.delete_at(loc)
     end
 
+    # Delete all occurences of the specified target
+    # @param obj object to delete
+    def delete(obj)
```

Comment:
@tpendragon I don't understand what you are asking.  Are you saying I should call a method named `#delete_target`, or I should rename my new method to `#delete_target`?